lib/beanstream/payments_api.rb

```diff
@@ -129,9 +129,10 @@ def make_payment(payment)
       val = transaction_post("POST", make_payment_url, Beanstream.merchant_id, Beanstream.payments_api_key, payment)
     end
     
-    def complete_preauth(transaciton_id, amount)
+    def complete_preauth(transaciton_id, amount, order_number = nil)
```

Comment:
While we're at it, we could fix this spelling error.  `transaction_id` instead of `transaciton_id`.